plugins/sms/applets/sms/twiml.php

```diff
@@ -60,10 +60,12 @@
 
 // Call flows still use the legacy <Sms> TwiML
 // for sending messages during calls.
-if(AppletInstance::getFlowType() == 'voice') {
+if(AppletInstance::getFlowType() == 'voice')
+{
 	$response->sms($sms, $message_opts);
 }
-else {
+else
```

Comment:
Isn't this the opposite of the PSR2 recommendation? 
http://www.php-fig.org/psr/psr-2/#if-elseif-else